### ISSUE TITLE
It is impossible to type cyrillic Б in profile/edit tags field

### ISSUE THREAD

**Issue by cmrd-senya** [diaspora/diaspora#6164]
If you try to type Б in russian keyboard layout to tags entry at the profile/edit page nothing happens, symbol doesn't appear. That possibly because it is on the same key that "," character, and somehow get blocked along with comma. I thought my key has been broken and started to disassemble my keyboard. And then just tried different input field.

**Comment by AntonTkachov** [comment]
I am not sure that using keypress will not cause any side effects, but I haven't find any problems during my research and manual testing. Jasmine also passed ok

**Comment by cmrd-senya** [comment]
I've just tested it on my development setup, and it works fine. However
on 0.5.1.1 at socializer.cc it doesn't. Probably it just has been solved
recently.

**Comment by jaywink** [comment]
OK so closing as fixed already, thanks.

**Comment by svbergerem** [comment]
@AntonTkachov Since you were trying to fix the issue with your PR: have you been able to reproduce the issue on the develop branch?

**Comment by AntonTkachov** [comment]
@svbergerem Yes. In fact I still can reproduce it on both develop branch and UA pod -> https://diaspora.scorry.net. I'm confused that @cmrd-senya haven't this issue on his development setup.

**Comment by AntonTkachov** [comment]
Since the issue had beed reopened I re-created pull request

**Comment by jaywink** [comment]
Sorry, misunderstood a comment :(

**Comment by cmrd-senya** [comment]
Ok, I tested last development version again on firefox and chromium.
Firefox has no issue, while chromium has. That's why I saw it on
production and didn't see on development - because I use firefox for
development and chromium for surfing. So we should probably rename topic
accordingly.
